examples/tracetest-datadog/tracetest/tracetest.config.yaml

```diff
@@ -33,6 +29,5 @@ telemetry:
 
 server:
   telemetry:
-    dataStore: otlp
     exporter: collector
     applicationExporter: collector
```

Comment:
can't you remove this as well?

Reply:
probablly could, but I want to focus this PR in just the specific change